src/main/java/pl/zimi/zadanka/Zad5.java

```diff
@@ -0,0 +1,119 @@
+package pl.zimi.zadanka;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.TreeSet;
+
+/*
+source:
+https://leetcode.com/problems/climbing-stairs/
+ */
+public class Zad5 {
+//    public void Node ( int destination,int currentPath){
+//        int left = currentPath+1;
+//        int right = destination+2;
+//    }
+    /*
+    key = destination pointer=current node
+    left/right child node
+     */
+    public static int climbStairs(int key) {
+        int solutionCounter = 0;
+        int startPointer=1;
+        int pointer =startPointer;
+        int left=1;
+        int right=2;
+
+        while ((pointer!=0)&&(pointer!=key)){
+
+
+        }
+
+
+
+        return solutionCounter;
+    }
+
+
+    public static int climbStairsTreeSet(int n) {
+        Set<String> treeSet = new TreeSet<>();
+        int sumary = 0;
+        int randomPath = 0;
+        String dummyPathString = "";
+        int allPasscount = 0;
+        int allFilteredCount = 0;
+        for (Integer k = 0; k < 100000000; k++) {
+            while (sumary < n) {
+                randomPath = (int) ((Math.random() * (2)) + 1);
```

Comment:
nie bardzo w sumie rozumiem logikę... to nie jest problem kodu tylko bardziej mój ale chyba będziemy musieli przegadać logikę tego kodu żebym zrozumiał